README.md

```diff
@@ -31,4 +31,4 @@ DDS directly instead of via the ROS2 abstraction.
 ## Known limitations
 
 Cyclone DDS doesn't yet fully implement the Lifespan, Deadline and some of the Liveliness QoS modes.
-Consequently these features of ROS2 are also not yet supported when using Cyclone DDS.
+Consequently these features of ROS2 are also not yet supported when using Cyclone DDS.
```

Comment:
There should not be a diff at all in this file. There might be a different newline or something. You should be able to `git checkout <commit> README.md` to wipe it if you want, using whatever commit is in your other PR.